addons/test_website_slides_full/tests/test_ui_wslides.py

```diff
@@ -12,56 +12,21 @@
 class TestUi(AccountTestInvoicingCommon, TestUICommon):
```

Comment:
Do we still need "AccountTestInvoicingCommon" ?

To answer your question on the PR, I would indeed clean and remove everything we don't need :slightly_smiling_face: